using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -538,9 +777,17 @@ ACPTarget.prefetchContent(prefetchArray, with: targetParameters, callback: { err
 {% endtab %}
 
 {% tab title="React Native" %}
-### prefetchContent
 
-Use this API to send a prefetch request to your configured Target server with the `ACPTargetPrefetchObject` array and the specified `ACPTargetParameters`. The callback is invoked when the prefetch is complete and, if successful, returns a null value. If the prefetch is not successful, an error message is returned.
+#### Syntax
+
+```javascript
+prefetchContent(prefetchObjectArray: Array<ACPTargetPrefetchObject>, parameters?: ACPTargetParameters): Promise<any>
+```
+* _prefetchObjectArray_ is an Array of `ACPTargetPrefetchObject` objects for various mbox locations.
+* _parameters_ is the configured `ACPTargetParameters` for the prefetch request.
+* A Promise object is returned which is resolved with true value or rejected with the error reason.
+
```

Comment:
Please reword the bulleted list as follows:
"* *prefetchObjectArray* is an Array of `ACPTargetPrefetchObject` objects for various mbox locations.
* *parameters* is the configured `ACPTargetParameters` for the prefetch request.
* A Promise object is returned and is resolved with true value or is rejected with the reason for the error.

Reply:
In one of the similar sections earlier in this document, "Array" was "array". Here it is "Array". Please make sure usage is consistent.